grants_tagger/__main__.py

```diff
@@ -205,49 +212,59 @@ def bioasq_mesh(
         cfg.read(config)
 
         input_path = cfg["preprocess"]["input"]
-        output_path = cfg["preprocess"]["output"]
+        train_output_path = cfg["preprocess"]["output"]
         mesh_tags_path = cfg["filter_mesh"].get("mesh_tags_path")
         test_split = cfg["preprocess"].getfloat("test_split")
 
-    # TODO: Refactor with preprocess_mesh
-    if test_split:
-        data_dir, data_name = os.path.split(output_path)
-        train_output_path = os.path.join(data_dir, "train_" + data_name)
-        test_output_path = os.path.join(data_dir, "test_" + data_name)
+    if verify_if_paths_exist(
+        [
+            input_path,
+            train_output_path,
+            label_binarizer_path,
+            test_output_path,
+        ]
+    ):
+        return
 
-        if os.path.exists(train_output_path) and os.path.exists(test_output_path):
-            print(
-                f"{train_output_path} and {test_output_path} exists. Remove them if you want to rerun."
-            )
+    temporary_output_path = train_output_path + ".tmp"
+    preprocess_mesh(input_path, temporary_output_path, mesh_tags_path=mesh_tags_path)
+    create_label_binarizer(temporary_output_path, label_binarizer_path)
+
+    if test_split:
```

Comment:
test_split is missing as a parameter to the CLI. also I think test_split should have a default value and only if test_output_path is None it should not split but not fixed on that. if you strongly feel otherwise keep as is.

Reply:
hm, ok, I favor that also, 0.1 I suppose would be the default value @nsorros ?

Reply:
0.1 for science and 0.01 for bioasq.